package.json

```diff
@@ -261,8 +261,8 @@
     "@datadog/browser-logs": "^5.8.0",
     "@datadog/browser-rum": "^4.49.0",
     "@department-of-veterans-affairs/component-library": "^42.1.1",
-    "@department-of-veterans-affairs/css-library": "^0.7.0",
-    "@department-of-veterans-affairs/formation": "11.0.5",
+    "@department-of-veterans-affairs/css-library": "^0.8.1",
+    "@department-of-veterans-affairs/formation": "11.0.6",
```

Comment:
@chriskim2311 Thank you for the detailed feedback! I'll leave a 👍 emoji on each of your comments after it's been addressed and ping your team when everything is ready for re-review (it might be a few days).

If I have any questions, I'll just quote reply below.

Reply:
> Public Websites
> 
> **Injected Header**
> 
> Screenshot
> Injected Header styles need to be updated in the styles.consolidated.css file https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/proxy-rewrite/manifest.json

@chriskim2311 We are not intending to change the injected header or footer styles at this point. Only VA.gov should be getting updated. I checked the injected header locally and compared it to prod and I'm not seeing any changes (which is expected).

Reply:
> **Breadcrumb wrapping on stories for tablet screen size**
> 
> Breadcrumb wrapping is off for tablet screen size compared to prod

@chriskim2311  I am seeing this wrapping behavior in prod too [on this page](https://www.va.gov/lovell-federal-health-care-va/stories/sculpture-dedicated-in-community-living-center-courtyard/). This is not the `va-breadcrumbs` component so I'm not sure the expected behavior but if it's already happening in prod, I don't think there's anything I should update as part of this PR.

![Screenshot 2024-06-13 at 1 02 27 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/872479/5e8c14b6-7ce7-4380-b52a-3105257394c4)

Reply:
@jamigibbs Here is a url for the vet-center spotlight! `/north-orange-county-vet-center/`

Reply:
Also for the breadcrumbs on facilities story pages, I am seeing a wrap in the breadcrumb at 800px size which differs in the PR to prod. Behavior is the same just the wrap happens on larger sizes for the PR.

Reply:
@chriskim2311 I think the wrapping behavior with `va-link` where the arrow icon is dangling in different viewports is something DST needs to address within the component itself. I created an issue to address it there instead of in this PR: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2938

Reply:
> Also for the breadcrumbs on facilities story pages, I am seeing a wrap in the breadcrumb at 800px size which differs in the PR to prod. Behavior is the same just the wrap happens on larger sizes for the PR.

@chriskim2311  What I think is happening here is that because the letters are a bit bigger, it's just shifting at a different place than before. That might be something we'd want to happen?

I can see in this comparison that the updated version (bottom image) will hit the right side sooner which is making it adjust down. I believe we'd want to allow that to continue to happen.

<img width="839" alt="Screenshot 2024-06-13 at 3 16 02 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/872479/ff0d11fe-28a5-47f5-a938-45de6fcb4ea0">

Reply:
@chriskim2311 Whenever you or @laflannery have time, you are welcome to re-review. This preview instance still looks like it's working at the moment: http://082361df76dba7d959e392fdaaf5bba0.review.vetsgov-internal/

Reply:
Thanks @jamigibbs I will start re-reviewing today!

Reply:
@jamigibbs We have re-reviewed:

1. Mega Nav
   - [This was discussed in slack](https://dsva.slack.com/archives/C06V7AAFVH7/p1718742362995919) and the VA.gov font is going to be increased to 16px
2. Facility Locator: Button text is off center
   - This is the only issue that we saw that was still outstanding   

### 2 new issues:
1. H3 heading styles
   - On some facility pages, when we have an H3 linked, the standard H3 styles (20px Bitter) don't appear to be applying, instead link styles (16.96px Source sans pro) are being used.

<details>
<summary>PR</summary>
http://082361df76dba7d959e392fdaaf5bba0.review.vetsgov-internal/boston-health-care/contact-us/

![Screenshot 2024-06-18 at 5 02 16 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/106678594/4a62ba5d-adc6-400b-87b1-25b5f4457c35)


</details>

<details>
<summary>Prod</summary>

![Screenshot 2024-06-18 at 4 58 36 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/106678594/b9150983-98d4-4f49-b692-25d7c117da50)


</details>

2. Column widths
   - There is an issue we found on [the facility pages](http://082361df76dba7d959e392fdaaf5bba0.review.vetsgov-internal/boston-health-care/locations/jamaica-plain-va-medical-center/) where at the 481px breakpoint, the content within columns runs into  each other. However I noticed that it's because the columns have no padding - and they never did.
   - We are using the `usa-width-two-thirds` and `usa-width-one-third` class to make the columns. Are these the proper classes for grid/columns? Or should we perhaps be using something else which is why this doesn't have any padding?

<details>
<summary>Screen shot</summary>
https://www.va.gov/boston-health-care/contact-us/

![Screenshot 2024-06-18 at 5 03 36 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/106678594/c9a49acf-f695-4cc9-9c28-7572e603816e)


</details>

Reply:
> We are using the `usa-width-two-thirds` and `usa-width-one-third` class to make the columns. Are these the proper classes for grid/columns? Or should we perhaps be using something else which is why this doesn't have any padding?

@laflannery About the column classes you've observed that don't have padding, I believe the preferred grid classes would be those that are prefixed with `vads-` like the ones demonstrated here: https://design.va.gov/foundation/layout/flexbox-grid

Reply:
@laflannery We have a [new review instance](http://6af3d7e56044a40ee9582a480d6ed24e.review.vetsgov-internal/) available now and it should include updates that address your last feedback items:

1. Mega Nav: marketing box font size
2. Facility Locator: Button text is off center
3. H3 heading styles (linked headings)

You're welcome to re-review whenever you have a chance.

Reply:
Thank you! Tagging in @thejordanwood since I am out tomorrow

Reply:
@jamigibbs It looks like the Facility Locator and the H3 heading styles were fixed. For the Mega Nav, I'm still noticing that the text in the housing assistance promo block doesn't fit within the space.

Reply:
@jamigibbs I just took a look at this and this issue is the title in the promos. Those are still at 1.06rem (16.96px) and they should also be set to 1rem (16px). That's why the text is being pushed too far down.

Reply:
@jamigibbs The Search button on facility locator is still off when viewing with Large or Very large font sizes, this is because it has a min width of pixels rather than relative font sizes. In order for font to resize appropriately, it must always have relative font sizes:

![Screenshot 2024-06-24 at 11 40 06 AM](https://github.com/department-of-veterans-affairs/vets-website/assets/106678594/e3f3ebda-840c-4bae-9373-8550d0d3c511)

Reply:
@laflannery It looks like [prod has a `min-width` already ](https://www.va.gov/find-locations/)so this might be an existing problem. Is this something that could be a separate issue for your team to look at?

![Screenshot 2024-06-24 at 10 48 19 AM](https://github.com/department-of-veterans-affairs/vets-website/assets/872479/3040f647-5810-46b5-aaea-165fbecada3c)

Reply:
I just confirmed this is all set at various font sizes, thank you!

I also confirmed the mega promo promo content fits at the standard font size, the other font sizes is a bigger issues we are handling separately, [slack thread for context](https://dsva.slack.com/archives/C06V7AAFVH7/p1719243755794759)

That should do it for Sitewide review @jamigibbs!